app/models/setting/general.rb

```diff
@@ -15,7 +15,7 @@ def self.default_settings
       self.set('max_trend', N_("Max days for Trends graphs"), 30, N_('Max trends')),
       self.set('db_pending_migration', N_("Should the `foreman-rake db:migrate` be executed on the next run of the installer modules?"), true, N_('DB pending migration')),
       self.set('db_pending_seed', N_("Should the `foreman-rake db:seed` be executed on the next run of the installer modules?"), true, N_('DB pending seed')),
-      self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy"), 60, N_('Proxy request timeout')),
+      self.set('proxy_request_timeout', N_("Max timeout for REST client requests to smart-proxy. Does not affect client timeout (hammer CLI, web browser)!"), 60, N_('Proxy request timeout')),
```

Comment:
I think this additional comment doesn't make a lot of sense. How about:
```suggestion
      self.set('proxy_request_timeout', N_("Timeout for HTTP requests from Foreman to Smart Proxy."), 60, N_('Proxy request timeout')),
```

Reply:
I am coming here from a customer case where we've fight with timeouts until we realized that it's the hammer what was actually timing out. I feel like I want to tell.

Reply:
That's why I said explicitly from Foreman to Smart Proxy, but I realize this is hard.

Perhaps the short description should be modified as well:
```suggestion
 self.set('proxy_request_timeout', N_("Timeout for HTTP requests from Foreman to Smart Proxy."), 60, N_('Smart Proxy request timeout')),
```